django-verdant/rca/static/rca/css/components/course-booking.less

```diff
@@ -0,0 +1,46 @@
+#course-booking-form {
+    overflow-x: auto;
+    margin-bottom: 20px;
```

Comment:
could use the `@gutter` var here if you fancied - https://github.com/torchbox/verdant-rca/blob/master/django-verdant/rca/static/rca/css/variables.less#L263